src/main/java/Cost.java

```diff
@@ -0,0 +1,25 @@
+public class Cost {
+
+    private static final int MINIMUM_PRICE = 1000;
+    private static final int LOTTO_PRICE = 1000;
+    private final int cost;
+
+    public Cost(int cost) {
+        this.cost = cost;
+        validate(cost);
```

Comment:
객체 생성 시점에 객체의 유효성 검증을 진행하고있군요 👍💯

객체를 생성하기 위해 인스턴스 변수에 값을 할당해주기 전에 먼저 유효성을 검증해주는게 조금 더 좋을 것 같아요. 😉
그러면 혹시나 유효하지 않은 경우에는 불필요한 할당을 조금이라도 하지 않을 수 있을 것같네요 🙂
```suggestion
        validate(cost);
        this.cost = cost;
```

Reply:
검증을 먼저하도록 수정했습니다. [a971c83](https://github.com/next-step/java-lotto-kakao/pull/12/commits/a971c8318e9c657cb37f18e4a9d556bfc535f997)